tests/installation/partitioning/msdos_partition_table.pm

```diff
@@ -22,10 +22,10 @@ sub run {
     my $partitioner = $testapi::distri->get_expert_partitioner;
     $partitioner->run_expert_partitioner;
     foreach my $disk (@{$test_data->{disks}}) {
-        $partitioner->create_new_partition_table({disk => $disk->{name},
-                table_type => $disk->{table_type}, accept_deleting_current_devices_warning => 1});
+        $partitioner->create_new_partition_table({name => $disk->{name},
```

Comment:
@OleksandrOrlov Can you confirm that this change is safe? From what I can see test is running against SLE 15 SP3 only, so we should be fine.

Reply:
This change is not only safe, but must. I also investigated that issue and this is one of the required fixes needed to be added here. So, should be ok.